src/main/java/org/folio/bulkops/processor/UpdatedEntityHolder.java

```diff
@@ -9,5 +9,6 @@
 @Builder
 public class UpdatedEntityHolder<T extends BulkOperationsEntity> {
   private T updated;
+  private T preview;
```

Comment:
Please, remove it. Task can be implemented without this change because preview for proposed updates is already implemented.